main/roc-toolkit/template.py

```diff
@@ -0,0 +1,52 @@
+pkgname = "roc-toolkit"
+pkgver = "0.4.0"
+pkgrel = 0
+hostmakedepends = [
+    "gengetopt",
+    "pkgconf",
+    "ragel",
+    "scons",
+]
+makedepends = [
+    "json-c-devel",
+    "libatomic_ops-devel",
+    "libltdl-devel",
+    "libpulse-devel",
+    "libsndfile-devel",
+    "libunwind-devel",
+    "libuv-devel",
+    "openssl-devel",
+    "speexdsp-devel",
+]
+pkgdesc = "Real-time audio streaming over the network"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "MPL-2.0"
+url = "https://roc-streaming.org"
+source = f"https://github.com/roc-streaming/roc-toolkit/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "60501dfdc8c0de385898dbd1eb8239d93ef962667ddb064ad796b887a41a1a46"
+hardening = ["vis", "cfi"]
+# check: needs cpputest
+# cross: scons, can't be  bothered
+options = ["!check", "!cross"]
+
+
+_scons_flags = [
+    "--disable-openfec",
+    "--disable-sox",
+    "--disable-alsa",
```

Comment:
what setup do you use that you get crackling in? is it pipewire on both sides using the plugins, or is one side using roc-send/recv?

if you're using the cli utils on either side then you might actually want to have alsa enabled so they are directly outputting and not going through an extra layer, or at least that's a guess

Reply:
sender is chimera linux with pipewire-roc, the receiver is android using roc droid, so i think what you're suggesting doesn't apply